lgsm/config-default/config-lgsm/sdtdserver/_default.cfg

```diff
@@ -134,7 +134,7 @@ stopmode="8"
 # 3: gamedig
 # 4: gsquery
 # 5: tcp
-querymode="5"
+querymode="2"
```

Comment:
This commit regressed #3764. Enabling monitor on 7 Days to Die continually restarts the server